src/styles/cards.scss

```diff
@@ -0,0 +1,122 @@
+.cards {
+  padding: 50px 40px 72px;
+  display: grid;
+  gap: 46px 48px;
+  justify-content: center;
+
+  @include set-columns(1);
+
+  @media (min-width: 488px) {
+    @include set-columns(2);
+  }
+
+  @media (min-width: 768px) {
+    @include set-columns(3);
+  }
+
+  @media (min-width: 1024px) {
+    @include set-columns(4);
+  }
+
+  &__card {
+    width: 200px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), fixed width for elements should be avoided where possible to maintain responsiveness. Consider using max-width or a percentage-based width.